api/onboarding/views.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+import requests
+from django.conf import settings
+from requests.exceptions import RequestException
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAdminUser
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.throttling import ScopedRateThrottle
+
+from integrations.lead_tracking.hubspot.services import (
+    create_self_hosted_onboarding_lead,
+)
+from onboarding.serializers import SelfHostedOnboardingSupportSerializer
+from users.models import FFAdminUser
+
+logger = logging.getLogger(__name__)
+
+
+SEND_SUPPORT_REQUEST_URL = "https://api.flagsmith.com/api/v1/onboarding/request/send/"
+
+
+@api_view(["POST"])
+@permission_classes([IsAdminUser])
+def send_onboarding_request_to_saas_flagsmith_view(request: Request) -> Response:
+    admin_user: FFAdminUser = request.user  # type: ignore[assignment]
+    organisation = admin_user.organisations.first()
+
+    if not organisation:
+        return Response(
+            {"error": "Please create an organisation before requesting support"},
```

Comment:
We have an ErrorSerializer that uses `"message"` as the key in the response [here](https://github.com/Flagsmith/flagsmith/blob/main/api/api/serializers.py#L4). I'm not sure how heavily it's used throughout the API, but we should at least try not to make things worse.